### ISSUE TITLE
Stable version for Pi 4

### ISSUE THREAD

**Issue by d-looper** [ccrisan/motioneyeos#2798]
### Preliminary Docs

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version

I am running motionEyeOS version: motionEyeOS 20200606 for pi 4
### Board Model

I am using the following board/model: Pi 4 8GB

### Camera

I am using the following type of camera: official PiNoIR

My camera model is: (insert your camera model, e.g. Logitech C290).

### Network Connection

My motionEyeOS unit is connected to the network via: WiFi/Ethernet

This is just a general question but when might we have a stable version of MotionEyeOS for the Pi 4? 

Ive gone through and updated everything on my 8GB version and using a known working official Pi NoIR camera (not a knock off), re-imaged motioneyeOS for the PI 4, working wpa_supplicant.conf (used on 2 zeros and a 3b). 

It will not boot with the base image without replacing the fixup4.dat and start4.elf files from another issue post. When it does boot, it will not recognize the camera using any of the options. To be sure i tested the camera on my 3b and the camera does work with motioneyeOS on the 3b, just not on the 4.

**Comment by starbasessd** [comment]
Did you try dev20201026?

**Comment by d-looper** [comment]
Where would i find that?

**Comment by starbasessd** [comment]
https://github.com/ccrisan/motioneyeos/releases/download/nightly-dev/motioneyeos-raspberrypi4-dev20201026.img.xz

**Comment by d-looper** [comment]
Thanks i will give that a try

**Comment by starbasessd** [comment]
I just confirmed it is working on my 8GB Pi4b, latest eeprom update:
April 29 2021

**Comment by starbasessd** [comment]
No need to update any files if using the latest rpi-eeprom and motionEyeOS dev20201026 (other than ssh and wpa_supplicant.conf if needed)

**Comment by d-looper** [comment]
That image worked for the boot and camera issues. One thing i did have to do is delete the camera that gets added by default and re-add it because it wouldnt let any settings be changed. I changed settings and rebooted, settings werent applied and cant change settings again.

**Comment by d-looper** [comment]
Are there any issues with Chrome in this build?

**Comment by d-looper** [comment]
I was able to get to the camera settings using firefox but that crashed as soon as i applied them. The settings were saved and applied though.

**Comment by starbasessd** [comment]
I am running out of hardware to check with. I will look into it shortly. There have been posted a few issues with Chrome but I only run the latest and greatest, and I seldom if ever have issues. If on Windows, check with Edge, also if you have the latest version, since it's chrome based.

**Comment by d-looper** [comment]
No rush. Just thought i would mention it. I have it working right now.

**Comment by starbasessd** [comment]
I just connected to a Pi3b+ running motioneye docker, chrome worked fine:
Version 92.0.4515.159 (Official Build) (64-bit) on Win10. 
Ditto to a i5 running Ubuntu 20.04 running motion 4.3.2, and motionEye 0.41. 
Pi4-8GB tied up for a bit. Might be late this evening before it's freed up, sorry.
Which version of Chrome are you running and what platform?

**Comment by d-looper** [comment]
I only just had this issue with the Pi 4 on the dev build. I have 2 Pi Zeros working fine (although i do have a follow up question on the zero which ill ask elsewhere). 
Im accessing it from a Windows 10 machine, Chrome Version 92.0.4515.159 (Official Build) (64-bit) - Latest version

**Comment by d-looper** [comment]
Its only an issue when accessing MotionEyeOS directly. I have MotionEye running on an ubuntu server and i can access the settings for the pi4 there without an issue

**Comment by starbasessd** [comment]
OK, will test and report later.

**Comment by starbasessd** [comment]
I just did a new, fresh install of dev20201026 to a 16GB card using Win32DiskImager, copied my wpa_supplicant.conf and ssh.txt files to /boot. Ejected and installed in a 8 GB Pi4B. Turned on, it booted normally. I then connected via Chrome Version 92.0.4515.159 (Official Build) (64-bit), FireFox 91.0.1 (64-bit), and Edge Version 92.0.902.84 (Official build) (64-bit). Added a camera, rebooted, and deleted the camera. All 3 performed as expected, no issues.
